deploy/examples/create-external-cluster-resources.py

```diff
@@ -506,16 +512,15 @@ def create_cephCSIKeyring_cephFSProvisioner(self):
         '''
         command: ceph auth get-or-create client.csi-cephfs-provisioner mon 'allow r' mgr 'allow rw' osd 'allow rw tag cephfs metadata=*'
         '''
-        cluster_name = self._arg_parser.cluster_name
-        cephfs_filesystem = self._arg_parser.cephfs_filesystem_name
         entity = "client.csi-cephfs-provisioner"
-        if cluster_name:
-            entity = "client.csi-cephfs-provisioner-{}".format(cluster_name)
         cmd_json = {}
         if self._arg_parser.restricted_auth_permission:
-            if cephfs_filesystem == "":
+            cluster_name = self._arg_parser.cluster_name
+            cephfs_filesystem = self._arg_parser.cephfs_filesystem_name
```

Comment:
I don't think the cephfs name is mandatory currently so let's not force it if `restricted_auth_permission` is on.

Reply:
so I was thinking to keep it mandatory for now,
because of this, I have to also adjust the node and provisioner secret and which may lead to different outputs for different cli inputs.
And the output may be used by dr/csi work, I guess which may affect the dependent code. @Madhu-1 can you provide some feedback.

maybe I can change it in follow-up PR @leseb?

Reply:
@leseb @Madhu-1, please look at it for an update?

Reply:
Isn't the output already adjusted if no FS is passed? We cannot change the current behavior and then revert it in a follow-up patch. We need to get it right in this PR. Thanks!

Reply:
if the `restricted_auth_permission` is set, then currently `cephfs_filesystem` was mandatory.

Okay, adding it.

Reply:
I have updated the changes in the second commit, which will be easy to review, 
And after that, I can squash the 2 commits, if needed.

Reply:
@leseb please have a look.